flink-connectors/flink-connector-elasticsearch-base/src/test/java/org/apache/flink/connector/elasticsearch/table/ElasticsearchDynamicSinkFactoryBaseTest.java

```diff
@@ -227,4 +227,42 @@ public void validateWrongCredential() {
                                                 "")
                                         .build()));
     }
+
+    @Test
+    public void validateInvalidTimeouts() {
```

Comment:
I think this test should be based in `ElasticsearchSinkBuilderBaseTest` because it verifies behaviour of the connector build and is not specific to the validation in the Table API.